lib/mappings/morph_to_wikidata.rb

```diff
@@ -0,0 +1,34 @@
+require 'everypolitician'
+
+module Mappings
+  class MorphToWikidata
+    def initialize(ep_slug:, ep_id_scheme:)
+      @ep_slug = ep_slug
+      @ep_id_scheme = ep_id_scheme
+    end
+
+    def to_h
```

Comment:
it's a little odd for all of these classes to have no other public methods other than `to_h` or `to_a`. This feels a little like a way to sidestep the question of what these methods actually return, or what the class represents. If that's because this is all just exploratory at this point, and we don't really *know* what to call them, then that might be quite a useful approach, but it would be good to at least leave a comment to note that this is hopefully only an interim measure.